### DISCUSSION TITLE
Badge with size?

### DISCUSSION THREAD

**Discussion by elalish** [okikio/bundle#29] [Q&A] [answered]
I'm looking to switch over from Bundlephobia, but one nice feature it has is a badge that includes the bundled size, which updates when a new npm version is released. I saw the Badges section in the readme, but it doesn't appear to include this functionality? Is this possible, or restricted based on how bundlejs is architected?

**Comment by okikio** [comment, marked as answer]
It's possible buts it's going to require a full on API, which is currently in development. 

> One last exciting development is that, I'm currently working on a platform agnostic library based on bundlejs that you can use anywhere including for vs code extensions, chrome extensions, deno, node, cloudflare workers, etc...

**Reply by wooorm** [reply]
I'd also be interested in a bundlephobia alternative for badges that I use in all my readmes.
I personally use badges through Shields.io: https://shields.io/.
I believe that they do some of that API for you?

**Reply by okikio** [reply]
@wooorm How would I go about setting up a badge API on shields.io? I can't find the docs on how to do it

**Reply by wooorm** [reply]
I don't exactly and I couldnt quickly find anything on mobile either unfortunately, but I'm sure something exists on their GH repo? 🤷‍♂️. I'd say, look at bundlephobia: https://github.com/badges/shields/tree/23c0406bedfc6930735e8f5ea75dfe34faf1f290/services/bundlephobia, and look at existing PRs adding things?

**Reply by okikio** [reply]
Thanks for the help 👍

**Comment by okikio** [comment]
@elalish Badges are now out along with the bundlejs API ![spring-easing's badge](https://deno.bundlejs.com/?q=spring-easing&badge=detailed)

**Reply by okikio** [reply]
Learn more about what the bundlejs API can do...https://twitter.com/jsbundle/status/1634455569078669313?s=20
